src/EnergyPlus/GeneralRoutines.hh

```diff
@@ -148,9 +148,9 @@ void CalcBasinHeaterPower(Real64 const Capacity,     // Basin heater capacity pe
 
 void TestAirPathIntegrity(EnergyPlusData &state, OutputFiles &outputFiles, bool &ErrFound);
 
-void TestSupplyAirPathIntegrity(EnergyPlusData &state, EnergyPlus::OutputFiles &outputFiles, bool &ErrFound);
+void TestSupplyAirPathIntegrity(EnergyPlusData &state, ZonePlenumData &dataZonePlenum, EnergyPlus::OutputFiles &outputFiles, bool &ErrFound);
 
-void TestReturnAirPathIntegrity(EnergyPlusData &state, OutputFiles &outputFiles, bool &ErrFound, Array2S_int ValRetAPaths);
+void TestReturnAirPathIntegrity(EnergyPlusData &state, ZonePlenumData &dataZonePlenum, OutputFiles &outputFiles, bool &ErrFound, Array2S_int ValRetAPaths);
```

Comment:
Does this require both `state` and `dataZonePlenum` args?  I know that zone plenum is now a part of the new state object already, so I'm wondering about why there is both.  In the long run, we will see the best implementation in place when the functions only require the very minimal amount of state they need, so if we can get away with just passing `dataZonePlenum` and one or two other parts of `state`, that would be great.